h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/matrix/SparseMatrixFactory.java

```diff
@@ -128,15 +127,14 @@ public static int initializeFromChunkIds(
         int[] _actualRows;
         
         InitializeCSRMatrixFromChunkIdsMrFun(
-            Frame f, int[] chunks, Vec[] vecs, Vec w, DataInfo di,
+            Frame f, int[] chunks, Vec w, DataInfo di,
```

Comment:
Minor suggestion: Since you're doing some meaningful renaming in this PR, I'd use longer names more meaningful for newcomers here as well.

Reply:
good idea, fixed